generators/generator-base.js

```diff
@@ -1838,6 +1838,33 @@ module.exports = class extends PrivateBase {
         return child;
     }
 
+    /**
+     * run a command using the configured Java build tool.
+     *
+     * @param {String} buildTool - maven | gradle
+     * @param {String} profile - dev | prod
+     * @param {Boolean} command - the command (goal/task) to run
+     * @param {Function} cb - callback when build is complete
+     */
```

Comment:
there's a `@return` missing

Reply:
Same issue with the method above, I corrected both